addon/components/ember-popper-base.js

```diff
@@ -89,7 +120,7 @@ export default class EmberPopperBase extends Component {
 
   @action
   update() {
```

Comment:
yeah, I see what you mean. I wanted to discourage using it, but having a mismatch in names could be even worse 😛 let's do that